interface/patient_file/summary/pnotes.php

```diff
@@ -173,6 +175,36 @@
 
 </div> <!-- end pnotes -->
 
+
+<script type="text/javascript">
+document.addEventListener('DOMContentLoaded', function() {
+    console.log("DOM fully loaded and parsed");
+
+    var knownDomains = ['open-emr.org'];
+    console.log("Known domains:", knownDomains);
+
+    function isDomainKnown(url) {
+        var hostname = (new URL(url)).hostname;
+        console.log("Checking hostname:", hostname);
+        return knownDomains.some(domain => hostname === domain || hostname.endsWith('.' + domain));
+    }
```

Comment:
This will break the feature IMO. User will have to accept most all redirects and is unacceptable.
I don't think I am misinterpreting code.